src/PlaywrightSharp.Chromium/ChromiumPage.cs

```diff
@@ -112,27 +121,32 @@ internal async Task InitializeAsync()
 
                 if (options.Geolocation != null)
                 {
-                    tasks.Add(_client.SendAsync("Emulation.setGeolocationOverride", options.Geolocation));
+                    tasks.Add(_client.SendAsync(new EmulationSetGeolocationOverrideRequest
+                    {
+                        Accuracy = (double)options.Geolocation.Accuracy,

```

Comment:
Can't we match types here?

Reply:
makes sense
I think `double` is good enough here
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/floating-point-numeric-types